saleor/static/dashboard-next/products/views/ProductVariant.tsx

```diff
@@ -104,42 +105,47 @@ export const ProductVariant: React.StatelessComponent<ProductUpdateProps> = ({
-                        variant={variant}
-                        header={
-                          variant ? variant.name || variant.sku : undefined
-                        }
-                        onBack={handleBack}
-                        onDelete={() => deleteVariant.mutate(variantId)}
-                        onImageSelect={handleImageSelect}
-                        onSubmit={(data: FormData) => {
-                          if (variant) {
-                            updateVariant.mutate({
-                              attributes: data.attributes
-                                ? data.attributes
-                                : null,
-                              costPrice: decimal(data.costPrice),
-                              id: variantId,
-                              priceOverride: decimal(data.priceOverride),
-                              quantity: data.quantity || null,
-                              sku: data.sku,
-                              trackInventory: true // FIXME: missing in UI
-                            });
+                      <>
+                        <WindowTitle
+                          title={maybe(() => data.productVariant.name)}
+                        />
+                        <ProductVariantPage
+                          errors={errors}
+                          saveButtonBarState={formSubmitState}
+                          loading={disableFormSave}
+                          placeholderImage={placeholderImg}
+                          variant={variant}
+                          header={
+                            variant ? variant.name || variant.sku : undefined
                           }
-                        }}
-                        onVariantClick={variantId => {
-                          navigate(
-                            productVariantEditUrl(
-                              encodeURIComponent(productId),
-                              encodeURIComponent(variantId)
-                            )
-                          );
-                        }}
-                      />
+                          onBack={handleBack}
+                          onDelete={() => deleteVariant.mutate(variantId)}
+                          onImageSelect={handleImageSelect}
+                          onSubmit={(data: FormData) => {
+                            if (variant) {
+                              updateVariant.mutate({
+                                attributes: data.attributes
+                                  ? data.attributes
+                                  : null,
+                                costPrice: decimal(data.costPrice),
+                                id: variantId,
+                                priceOverride: decimal(data.priceOverride),
+                                quantity: data.quantity || null,
+                                sku: data.sku,
+                                trackInventory: true // FIXME: missing in UI
```

Comment:
Btw. is it still missing in UI? :)

Reply:
Yes... I think we can add checkbox in 'Inventory' card, like "Charge taxes for this item" in "Pricing" card.

<img width="1121" alt="zrzut ekranu 2018-11-13 o 17 09 05" src="https://user-images.githubusercontent.com/6833443/48426334-e0437d80-e766-11e8-9337-a71ef2fc591d.png">